e2e-polybft/e2e/consensus_test.go

```diff
@@ -595,3 +619,22 @@ func TestE2E_Consensus_CustomRewardToken(t *testing.T) {
 	require.NoError(t, err)
 	require.True(t, validatorInfo.WithdrawableRewards.Cmp(big.NewInt(0)) > 0)
 }
+
+func queryNativeERC20Metadata(t *testing.T, funcName string, abiType *abi.Type, relayer txrelayer.TxRelayer) interface{} {
+	t.Helper()
+
+	valueHex, err := ABICall(relayer, contractsapi.NativeERC20Mintable,
+		ethgo.Address(contracts.NativeERC20TokenContract),
+		ethgo.ZeroAddress, funcName)
+	require.NoError(t, err)
+
+	valueRaw, err := hex.DecodeHex(valueHex)
+	require.NoError(t, err)
+
+	var decodedResult map[string]interface{}
+
+	err = abiType.DecodeStruct(valueRaw, &decodedResult)
+	require.NoError(t, err)
+
+	return decodedResult["0"]
+}
```

Comment:
Put it in the `e2e-polybft/e2e/helpers_test.go` maybe?

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1623/commits/f00d6b8e7a08562b04fb8d9ef70d0421dbccd298